central/test-framework/org.jboss.tools.central.reddeer/src/org/jboss/tools/central/reddeer/utils/Utils.java

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.central.reddeer.utils;
+
+import static org.junit.Assert.fail;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+/**
+ * 
+ * @author zcervink@redhat.com
+ *
+ */
+
+
+public class Utils {
```

Comment:
I would suggest more meaningful class name like CentralUtils, CentralTestUtils or something similar. Utils is quite general.

Reply:
I have renamed it to CentralUtils.